tests/subsys/net/openthread/rpc/common/test_rpc_env.h

```diff
@@ -44,3 +45,17 @@
 #define EXT_PAN_ID    0x48, INT_SEQUENCE(OT_EXT_PAN_ID_SIZE)
 #define STEERING_DATA 0x50, INT_SEQUENCE(OT_STEERING_DATA_MAX_LENGTH)
 #define TLVS          0x58, 0xFE, INT_SEQUENCE(OT_OPERATIONAL_DATASET_MAX_LENGTH)
+#define NWK_KEY       0x50, INT_SEQUENCE(OT_NETWORK_KEY_SIZE)
+#define LOCAL_PREFIX  0x48, INT_SEQUENCE(OT_IP6_PREFIX_SIZE)
+#define PSKC          0x50, INT_SEQUENCE(OT_PSKC_MAX_SIZE)
+#define TIMESTAMP     CBOR_UINT64(0x0123456789abcdef), CBOR_UINT16(0x1234), CBOR_FALSE
+#define SECURITY_POLICY                                                                            \
+	CBOR_UINT16(0x9876), CBOR_TRUE, CBOR_FALSE, CBOR_TRUE, CBOR_FALSE, CBOR_TRUE, CBOR_FALSE,  \
+		CBOR_TRUE, CBOR_FALSE, CBOR_TRUE, 0x07
+#define COMPONENTS                                                                                 \
```

Comment:
nit: the name seems to generic for the common file. `DATASET_COMPONENTS` would give more hint to a future reader.